javascript-algorithms-and-data-structures/learn-intermediate-algorithmic-thinking-by-building-a-dice-game/index.js

```diff
@@ -1,272 +1,265 @@
+// targets the radio buttons and the spans
 const scoreInputs = document.querySelectorAll("#score-options input");
 const scoreSpans = document.querySelectorAll("#score-options span");
-const diceDivs = document.querySelectorAll("#dice > div");
+
+// square boxes that display the dice values
+const diceButtons = document.querySelectorAll("#dice > button");
```

Comment:
```suggestion
const diceButtons = document.querySelectorAll(".die");
```

Updated query selector to reflect new HTML.

Reply:
added